### PR TITLE
Task 1, Task2, Task3, Task4, Task5

### PR DESCRIPTION
**Pull request by shashankp0402**

### CODE REVIEW

**Comment by swathinelakurthi** [comment]
Activities are not named properly.
Properly specify the Task name.

File: SourceCode/2023/August2023/Shashank/Main.xaml
Comment (vajrang-b) [inline SourceCode/2023/August2023/Shashank/Main.xaml:1]: File available in in correct location

**Comment by vajrang-b** [comment]
image
task 2

**Comment by vajrang-b** [comment]
task3

default activity names no annotations

**Comment by vajrang-b** [comment]
task 4
used string and int instead of using generic variables

image

**Pull request merged**